src/client/fuse_ll.cc

```diff
@@ -1252,6 +1258,9 @@ int CephFuse::Handle::init(int argc, const char *argv[])
   auto fuse_debug = client->cct->_conf.get_val<bool>(
     "fuse_debug");
 
+  entry_timeout = client->cct->_conf.get_val<int64_t>("fuse_entry_timeout");
+  attr_timeout = client->cct->_conf.get_val<int64_t>("fuse_attr_timeout");
```

Comment:
We need to make this to be setable while the fuse daemon is running. Here in `init()` if you want to disable it you must remount it. Please refer to `Client::handle_conf_change()`.